react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,31 @@
+#root {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  font-family: sans-serif;
+}
+
+form {
+  display: flex;
+  align-items: center;
+  margin-top: 6px;
+}
+
+p {
+  color: red;
+  font-size: 13px;
+  margin: 0;
+  padding-top: 3px;
+}
+
+.fas {
+  margin-left: 10px;
+}
+
+.fas.fa-check {
+  color: green;
+}
```

Comment:
Avoid styling 3rd-party CSS classes. Instead add your own class to use alongside them.